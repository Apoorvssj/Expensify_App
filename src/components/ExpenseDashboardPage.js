import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';
import ExpensesSummary from './ExpensesSummary';

//using shorthand(impicit return)
const ExpenseDashboardPage = () => (
    <div>
       <ExpensesSummary />
       <ExpenseListFilters />
       <ExpenseList />
    </div>
   );

export default ExpenseDashboardPage;