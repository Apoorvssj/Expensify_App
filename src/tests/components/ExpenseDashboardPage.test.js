import React from 'react';
import {shallow} from 'enzyme';
import toJSON from 'enzyme-to-json';
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage'

test('should render ExpenseDasboardPage correctly', () => {
    const wrapper = new shallow(<ExpenseDashboardPage />);
    expect(toJSON(wrapper)).toMatchSnapshot();
});