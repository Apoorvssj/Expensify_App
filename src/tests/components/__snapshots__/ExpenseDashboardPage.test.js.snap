// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseDasboardPage correctly 1`] = `
<div>
  <Connect(ExpensesSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
